provisionersdk/proto/provisioner.proto

```diff
@@ -27,6 +27,8 @@ message RichParameterOption {
 
 // RichParameter represents a variable that is exposed.
 message RichParameter {
+    reserved 14;
```

Comment:
Should we reserve the name as well? I doubt it will be a problem, but I'm overly cautious 😄.

Reply:
Oh yeah, that's a safer approach 👍